system-parachains/collectives/collectives-polkadot/src/fellowship/tracks.rs

```diff
@@ -110,13 +119,32 @@ const PROMOTE_MIN_SUPPORT: pallet_referenda::Curve = pallet_referenda::Curve::Li
 	ceil: Perbill::from_percent(100),
 };
 
+const FAST_PROMOTE_MAX_DECIDING: u32 = 10;
+const FAST_PROMOTE_DECISION_DEPOSIT: Balance = 5 * DOLLARS;
+const FAST_PROMOTE_PREPARE_PERIOD: BlockNumber = 0;
+const FAST_PROMOTE_DECISION_PERIOD: BlockNumber = 7 * DAYS;
```

Comment:
Dunno if this is too short? The origin is already "faster" since it can bypass the promotion period.  
On the other hand, with the high turnout it will rather error on the side of rejecting so maybe its fine.

Reply:
The main issue is having to wait for years to get to a correct level.

If this can bypass the promotion min period, I don't see the need to have it faster than the "normal" 30 days, I don't expect members to find 30 days too slow..

Reply:
> The main issue is having to wait for years to get to a correct level.

Yeah that needs to be fixed by making a call in Core Fellowship that will respect this origin.

> Dunno if this is too short?

I think it is OK, could also go in the middle with something like 14 days.

Reply:
No strong opinions, so changed to 30 days after internal discussion.